src/components/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -217,17 +221,23 @@ const proceed = () => {
 }
 
 onMounted(() => {
-  document.addEventListener('keydown', handleKeydown)
+  if (typeof document !== 'undefined') {
+    document.addEventListener('keydown', handleKeydown)
+  }
 
   console.warn("The Kongponents 'KModalFullscreen' component is deprecated and will be removed in the next major release.\nWe suggest using 'KModal' component instead.\nDocs: https://kongponents.konghq.com/components/modal.html#fullscreen")
 })
 
 onBeforeUnmount(() => {
-  document.removeEventListener('keydown', handleKeydown)
+  if (typeof document !== 'undefined') {
+    document.removeEventListener('keydown', handleKeydown)
+  }
 })
 
 onUnmounted(() => {
-  document.body.style.overflow = ''
+  if (typeof document !== 'undefined') {
+    document.body.style.overflow = ''
+  }
```

Comment:
`onBeforeUnmount` and `onUnmounted` are only ever called in the client, so these checks shouldn't be needed